test/test_helper.rb

```diff
@@ -14,6 +14,14 @@ class ActiveSupport::TestCase
 class ActionDispatch::IntegrationTest
   include Capybara::DSL
 
+  def create_category_and_items(num)
+    num.times do |i|
+      category_one = Category.create(name:"coffee#{i}")
+      category_one.items.create(title: "pour over#{i}", description: 'it taste really good', price: 4.00, image: "google.com")
+      category_one.items.create(title: "drip#{i}",      description: 'it taste good',         price: 4.00, image: "google.com")
```

Comment:
Line is too long. [127/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.